config/routes.rb

```diff
@@ -62,6 +62,8 @@
   match '/edit/email' => 'users/edit_email#update', via: [:patch, :put]
   get '/edit/phone' => 'users/edit_phone#edit'
   match '/edit/phone' => 'users/edit_phone#update', via: [:patch, :put]
+  get '/settings/password' => 'users/edit_password#edit'
+  patch '/settings/password' => 'users/edit_password#update'
```

Comment:
Because it clashes with the Devise Passwords Controller path, even though they're not the same. I couldn't get it to work, so I created a new route. We'll probably rename the other paths to `settings` or whatever else we decide on. If we really want to keep using `edit/#{attribute}`, then we can spend extra time later getting it to work with Devise.